src/Servers/HttpSys/src/IHttpSysRequestTimingFeature.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Server.HttpSys;
+
+/// <summary>
+/// This exposes the Http.Sys HTTP_REQUEST_TIMING_INFO.
+/// </summary>
+public interface IHttpSysRequestTimingFeature
+{
+    /// <summary>
+    /// Gets all http.sys timing timestamps
+    /// </summary>
+    /// <remarks>
+    /// These timestamps were originally retrieved using <see cref="System.Diagnostics.Stopwatch.GetTimestamp"/> and the timestamp frequency can be obtained via <see cref="System.Diagnostics.Stopwatch.Frequency"/>.
```

Comment:
You are correct, http.sys calls QueryPerformanceCounter which is also what Stopwatch calls today. Technically Stopwatch could change but I doubt it. I was going to ask if it's better to refere to QueryPerformanceCounter or Stopwatch here. I choose Stopwatch since it's easy to link to but I don't have a strong opinion here.

Reply:
QueryPerformanceCounter is more accurate.